rules-reviewed/eap7/eap6/tests/data/data-hsearch/HsearchUtil.java

```diff
@@ -107,13 +111,13 @@ private void callConstructors(){
         EntityDescriptor entityDescriptor = new EntityDescriptor();
         entityDescriptor.setCacheInMemory(new HashMap());
         entityDescriptor.getCacheInMemory();
-        NumericFieldMapping numMapping = new NumericFieldMapping("test", PropertyDescriptor property, entityDescriptor, new SearchMapping()));
-        
+        NumericFieldMapping numMapping = new NumericFieldMapping("foo", property, entityDescriptor, new SearchMapping());
+
```

Comment:
you caught the issue here, but I made another mistake here too as the deprecated constructor is 
`NumericFieldMapping(property, entityDescriptor, new SearchMapping());` without the first `java.lang.String` argument :(
But the condition are not still met. I will need to find out why?.